### ISSUE TITLE
Custom Virtual Keyboard invalid type error in npm

### ISSUE THREAD

**Issue by pratyusharma** [arnog/mathlive#772]
## Description
Firstly thanks a lot for putting together this package. This is a great library and extremely responsive to mobile screens as well. 

**Problem faced**: 
- I load a mathlive field on the page with custom virtual keyboards named -- _math_ and _chem_. Everything works fine when I run the project and the user can interact with both these virtual keyboards as well. But the project is not able to be built because the keys inside _customVirtualKeyboards_ expect _strings_ and I have assigned an object to it. 

- The other problem faced is that, ```{ignoreSpacebarInMathMode: false}``` doesn't seem to allow entering a space in the mathlive field. I tried on Safari as well as Chrome.

## Steps to Reproduce

This is the code I'm using for initialising the mathlive keyboard and creating the virtual keyboards:
```typescript
import {makeMathField} from 'mathlive';

addCustomKeyboard(){
    var mfe = document.getElementById("mfe")
    makeMathField(mfe, {
      smartMode: false,
      ignoreSpacebarInMathMode: false,
      virtualKeyboardMode: 'manual',
      customVirtualKeyboardLayers: {
      'layer-1': {
          rows: [
             ......
          ],
        },
      'layer-2': {
          rows: [
             ......
          ],
        },
      },
      customVirtualKeyboards: {
        "math" : { //(Error shows here)
          label: 'X<sup>n</sup>',
          tooltip: 'Custom math',
          layer: 'layer-1',
        },
        "chem": { //(Error shows here)
          label: 'H<sub>2</sub>O',
          tooltip: 'Custom chem',
          layer: 'layer-2',
        },
      },
      virtualKeyboards: 'math chem functions symbols greek',
    })
  }
```

The error that is shown for "math" and "chem" is
```typescript
error TS2322: Type '{ label: string; tooltip: string; layer: string; }' is not assignable to type 'string'.ts(2322)
options.d.ts(335, 9): The expected type comes from this index signature.
```

The option.d.ts(335, 9) contains the following code:
```typescript
export declare type VirtualKeyboardOptions = {
......
    customVirtualKeyboardLayers: {
        [layerName: string]: string | VirtualKeyboardLayer;
    };
    customVirtualKeyboards: {
        [layerName: string]: string; //--> options.d.ts(335, 9)
    };
......
};
```
### What I tried
- Because of this the project fails to build but surprisingly works exactly as required.
I tried passing the object through ```JSON.stringify()``` before assigning it to the key, but then the custom virtual keyboard stopped working. I tried to use a ```new MathfieldElement``` object instead of using the ```makeMathField()``` function and added the custom virtual keyboards to it but that showed the same error.

- For the ```{ignoreSpacebarInMathMode: false}``` I tried creating a ```<math-field></math-field>``` with the attribute ```ignore-spacebar-in-math-mode="false"``` but it still didn't allow the user to input a space. I tried the spacebar key on the default keyboard but that didn't allow the user to enter a space either.

### Environment
**Angular**
Angular CLI: 9.1.13
Node: 10.16.0
OS: darwin x64

**NPM Package**
"mathlive": "^0.59.0"

**Operating System**
macOS 10.14

**Browser** 
Safari 12.0

**Comment by arnog** [comment]
I believe the first issue is fixed in mathlive@0.60.0.

The second issue is also fixed in mathlive@0.60.0. The `ignoreSpacebarInMathMode` option has been removed and replaced with `mathModeSpace` (see #859).

**Comment by LinhTT46** [comment]
Hi @pratyusharma, I just started looking for creating a custom virtual keyboard for Chem. I want to **enter** elements in the periodic table with and without virtual keyboard. However, I can not enter alphabet following the mchem style by physical keyboard. 
How do you setup for the mchem input? and Do you have a demo (or example code) for the Chem keyboard layer above somewhere per chance?
